### DISCUSSION TITLE
Custom hotkeys

### DISCUSSION THREAD

**Discussion by tosm05** [Eugeny/tabby#6816] [Q&A]
Hi, is it possible to create custom hotkeys, for example to insert a specific string?

**Comment by Eugeny** [comment]
Check out the quick-cmds plugin.

**Reply by tosm05** [reply]
Thanks for your reply. When trying to install the plugin in the app I'm receiving an error message which is: [object Object] I've tried dropping the quick-cmds files into the Plugins Folder but I also can't get that to work, on reopening the app all plugins are disabled. Do you have any advice please?
